src/main/java/core/basesyntax/service/FileReaderService.java

```diff
@@ -0,0 +1,8 @@
+package core.basesyntax.service;
+
+import java.io.File;
+import java.util.List;
+
+public interface FileReaderService {
+    List<String> readFromFile(File file);
```

Comment:
what do you think of passing a String as a parameter here? In this case, you won't need to make a file from path in Main

Reply:
same for file IO methods